### PR TITLE
Foundation 6 cheat sheet

### PR DESCRIPTION
**Pull request by jppope**
**What does your Instant Answer do?**

**What problem does your Instant Answer solve (Why is it better than organic links)?**

Makes it easier to use Foundation 6.

**What is the data source for your Instant Answer? (Provide a link if possible)**

foundation.zurb.com

**Why did you choose this data source?**

because Foundation is Effing awesome... and it is significantly better than bootstrap.

**Are there any other alternative (better) data sources?**

no
**What are some example queries that trigger this Instant Answer?**

I don't know what that is... but you should just have all queries related to foundation trigger it

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**

Developers... the good ones

**Is this Instant Answer connected to a DuckDuckHack Instant Answer idea?**

Yes. Los angeles Represent

**Which existing Instant Answers will this one supercede/overlap with?**

nope

**Are you having any problems? Do you need our help with anything?**

yea we still need to get a form / non technical version of the instant answers built

**Where did you hear about DuckDuckHack? (For first time contributors)**

I run the LA group

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**

dude? it looks awesome!

-----
IA Page: https://duck.co/ia/view/foundation_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@jppope Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

While the team and the community review your submission, we'd like to make sure you look good! If you'd like a free DuckDuckGo tshirt, please send over the following info to merch@duckduckgo.com, and be sure to put "Dev Merch" in the subject line:

Name:
Street Address:
City:
State / Province:
Country:
Phone (Non-US Orders only): 
T-shirt size:
GitHub ID:
Link to this Pull Request:

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

File: share/goodie/cheat_sheets/classic_cocktails.json [hunk 1]
```diff
@@ -0,0 +1,144 @@
+{
```
Comment (codenirvana) [inline share/goodie/cheat_sheets/classic_cocktails.json]: @jppope This shouldn't be here. You should use different branches for different PRs

Comment (jppope) [inline share/goodie/cheat_sheets/classic_cocktails.json]: Sorry... I thought that was already added from my last pull requests. What
needs to change with that one?

File: share/goodie/cheat_sheets/foundation.json [hunk 2]
```diff
@@ -0,0 +1,235 @@
+
+   {"id" : "CSS_cheat_sheet",
```
Comment (codenirvana) [inline share/goodie/cheat_sheets/foundation.json]: change this to ```foundation_cheat_sheet```

File: share/goodie/cheat_sheets/foundation.json [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+   "name" : "Foundation 6 Cheat Sheet",
+   "description" : "Basic Foundation shortcuts and love",
+   "metadata" : {
+      "sourceName" : "Zurbs Foundation 5",
```
Comment (codenirvana) [inline share/goodie/cheat_sheets/foundation.json]: Change this to just ```Zurb``` or ```Zurb Foundation```

File: share/goodie/cheat_sheets/foundation.json [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+   "section_order" : [
+	  "Content Delivery Network",
+      "Standard Grid Large",
+      "Standard Grid Medium",
+      "Standard Grid Small",
+      "Buttons",
+      "Random Utilities",
+      "shortcut Links"
+   ],
+   "description" : "Foundation Shortcuts",
```
Comment (codenirvana) [inline share/goodie/cheat_sheets/foundation.json]: Remove this second description.

File: share/goodie/cheat_sheets/classic_cocktails.json (hunk 1 above)
Comment (codenirvana) [inline share/goodie/cheat_sheets/classic_cocktails.json]: You need to delete the cocktails cheat sheet from this PR.
Make sure to backup your both cheat sheets first and then do,
```
git rm share/goodie/cheat_sheets/classic_cocktails.json
git commit -am 'deleted classic_cocktails.json'
```

Also, you should use different branches for different PRs. Its recommended not to make any changes at the ```master``` branch (like in this case). Use this type of workflow:
```
[fixing-bug-1] $ git checkout master
[master] $ git pull upstream master
[master] $ git branch new-cheat-sheet
[master] $ git checkout new-cheat-sheet
[new-cheat-sheet] $
```

File: share/goodie/cheat_sheets/foundation.json [hunk 5]
```diff
@@ -0,0 +1,235 @@
...
+	   "Content Delivery Network": [
+		   {
+            "val" : "Coming Soon",
+            "key" : "CSS"
+         },
+          {
+            "val" : "Coming Soon",
+            "key" : "Javascript"
+         },
+	   ]
```
Comment (codenirvana) [inline share/goodie/cheat_sheets/foundation.json]: You missed the ```,```  here

**Comment by codenirvana** [comment]
There's some indentation problems in JSON file.
Use tools like ```JSONLint``` or ```JSONFormatter``` to fix that problem for you!

**Comment by jophab** [comment]
@jppope The source url should be of some cheatsheet for foundation. And the source name should reflect the domain of  the url.

**Comment by jophab** [comment]
@jppope I think its good to start the val field with a capital letter

**Comment by jppope** [comment]
So I Added the CDN after the fact... CDNJS hasn't updated to foundaton 6
yet. Good Catch. Thank you! I linted the hell out of it so it should be
good now.

I purposely had the classes as lower case and the Explanation as Uppercase.

The URL source is foundation's documentation ... it wouldn't make any sense
that there would be cheat sheets for foundation 6 since it was deployed
Thursday. Plus... if there are already cheat sheets and documentation we
shouldn't be making cheat sheets for the results we should just link to the
results.

I'm pulling the classic cocktails out for this next pull request

File: share/goodie/cheat_sheets/foundation.json [hunk 6]
```diff
@@ -0,0 +1,241 @@
+{
+    "id": "CSS_cheat_sheet",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/foundation.json]: please change to `foundation_cheat_sheet`

**Comment by moollaza** [comment]
@jppope Ping! I'm just checking in to see if you're still interested in finishing this one up? We'd love to see this go live!

There's absolutely no rush, just wanted to make sure you're not stuck on anything.

**Comment by jppope** [comment]
Yes I would very much like to move forward... I just don't know what the
issues were after I made the modifications.

**Comment by moollaza** [comment]
@jppope the `id` needs to be changed: `foundation_cheat_sheet`

File: share/goodie/cheat_sheets/foundation.json [hunk 7]
```diff
@@ -0,0 +1,241 @@
...
+    "name": "Foundation 6 Cheat Sheet",
+    "description": "Foundation Shortcuts",
+    "metadata": {
+        "sourceName": "Zurbs Foundation 5",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/foundation.json]: Let's change this to `Zurb`, typically it should represent the domain name

File: share/goodie/cheat_sheets/foundation.json [hunk 8]
```diff
@@ -0,0 +1,241 @@
+{
+    "id": "CSS_cheat_sheet",
+    "name": "Foundation 6 Cheat Sheet",
+    "description": "Foundation Shortcuts",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/foundation.json]: We should make this more descriptive, so it explains the elements of the cheat sheet (high level), or remove it

File: share/goodie/cheat_sheets/foundation.json [hunk 9]
```diff
@@ -0,0 +1,241 @@
+{
+    "id": "CSS_cheat_sheet",
+    "name": "Foundation 6 Cheat Sheet",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/foundation.json]: Please remove "Cheat Sheet"

**Comment by moollaza** [comment]
@jppope I've left some more feedback that needs to be implemented. Once things are updated please let us know by leaving a comment, so we can deploy this to Beta and review this again.

If you have any questions, feel free to ask them here!

**Comment by jppope** [comment]
Ok... I'll review the feedback

**Comment by moollaza** [comment]
> Ok... I'll review the feedback
@jppope :+1:

**Comment by moollaza** [comment]
continuing in #2100

**Pull request closed without merging**